cluster/src/cluster_impl.rs

```diff
@@ -174,6 +175,54 @@ impl Inner {
 
         Ok(resp)
     }
+
+    async fn open_shard(&self, req: &OpenShardRequest, _opts: OpenShardOpts) -> Result<()> {
+        let shard_info = req.shard.as_ref().context(OpenShard {
+            shard_id: 0u32,
```

Comment:
Replace by a const? Such as `DEFAULT_SHARD_ID`?

Reply:
Actually, this is the error field of `OpenShard` and in this case, the error denotes the shard is not found in the request, so it is wired to set it as DEFAULT_SHARD_ID.

Reply:
Ok.